cranelift-codegen/meta/src/cdsl/ast.rs

```diff
@@ -630,7 +574,7 @@ macro_rules! def_rhs {
 
     // inst.type(a, b, c)
```

Comment:
follow-up nit: you should also add support for `inst.t1.t2(a, b, c)`, to support producing instructions such as `fcvt_from_uint`

Reply:
added this to https://github.com/CraneStation/cranelift/issues/734